src/app/fdctl/Local.mk

```diff
@@ -40,7 +41,7 @@ $(call add-objs,configure/xdp,fd_fdctl)
 $(call add-objs,configure/ethtool,fd_fdctl)
 $(call add-objs,configure/workspace,fd_fdctl)
 
-$(call make-bin-rust,fdctl,main,fd_fdctl fd_disco fd_flamenco fd_quic fd_tls fd_ip fd_reedsol fd_ballet fd_waltz fd_tango fd_util solana_validator)
+$(call make-bin-rust,fdctl,main,fd_fdctl fd_choreo fd_disco fd_flamenco fd_funk fd_quic fd_tls fd_ip fd_reedsol fd_ballet fd_waltz fd_tango fd_util solana_validator, $(SECP256K1_LIBS))
```

Comment:
Guard these changes with `FD_HAS_NO_SOLANA` same as fddev?

Reply:
(Then move the FD_HAS_SECP256K1 check to the same place

Reply:
done